comp120-evaluation.tex

```diff
@@ -0,0 +1,57 @@
+
+If I have not developed this skill in preparation for the forthcoming collaborative projects, I will struggle to review my peers' contributions and understand the code I am contributing to. This is equally important if I am going to collaborate with others professionally as an independent games developer.
+
+In order to develop this skill, I will ensure that I read peers' code each week and attempt to predict its outcome before running the program. I will review how often I was able to accurately understand what the code would do after two months.
+
+\section{Time Management}
+My perfectionistic nature combined with being out of education and work for a long time has caused me to struggle to manage my time effectively. Rather than not spending enough time working, the most significant issue for me is spending more time than necessary perfecting a single project whilst neglecting others. I have also noticed that I get distracted by adding unnecessary features to projects. 
+
+It is crucial that I develop my ability to manage my time in preparation for the increased workload as the course progresses. Similarly, it is important that I learn when to consider something done, in order to focus on other tasks. This will also be important if I become an independent games developer, as it will allow games to be released rather than being endlessly worked on. After extensively using Trello with my Kivy project, I have noticed that it helps with this aspect. I think that this is because it breaks the project down and allows me to see when I have completed all of the tasks. It also ensures that I focus on what's in the sprint plan, rather than adding unnecessary features.
+
+I will tackle this issue by creating a flexible study plan at the beginning of each week. I will also set up a Trello board for each project and utilise sprint planning. At the end of the second term, I will assess whether this allowed me to comfortably meet assignment deadlines.
+
+\section{Communication}
+I have had immense difficulties with communication, primarily due to anxiety. The aspect of this that has affected my learning the most is a reluctance to ask for feedback, help and clarification when needed. Initially, I avoided asking for help and attempted to solve every problem myself. Subsequently, this resulted in a reduction of productivity and high stress levels close to the deadline.
+
+It is important that I confront this issue, as I waste a lot of time pondering uncertainties or being stuck. Consequently, my struggles in this area also adversely affect the effectiveness of my time management. Tackling this is crucial if I am to keep up as the workload increases. Additionally, if I were to fulfill my career goal of becoming an independent games developer, I must learn to communicate effectively and be willing to ask for help, as it is not possible to tackle every problem alone.
```

Comment:
Instead of 'tackle every problem alone', I would propose some specific examples. Such as, getting help to learn new tools and/or working with the open source community.